### PR TITLE
Feature/join managed joined group pages

### LINKED ISSUE #363: Join managed and joined group page
## Motivations
Join the `/#/groups/managed` and `/#/groups/joined` pages to the page under `/#/groups/mine`.

## Subtasks
* Just merge the two pages (and delete the previous ones), so put in it:
  - a title: "My groups", subtitle: All the groups you have joined and you manage.
  - pending invitations
  - pending requests
  - groups you joined
  - groups you managed

### PR DESCRIPTION
**Pull request by Iloveall**
Fixes #363

### CODE REVIEW

File: src/app/modules/group/pages/my-groups/my-groups.component.html [hunk 1]
```diff
@@ -1 +1,27 @@
-<p>my-groups works!</p>
+<div class="group-info-container">
+  <div class="group-info-header">
+    <span class="group-title" i18n>Joined Groups</span>
```
Comment (smadbe) [inline src/app/modules/group/pages/my-groups/my-groups.component.html:3]: "My groups"

File: src/app/modules/group/pages/my-groups/my-groups.component.html [hunk 2]
```diff
@@ -1 +1,27 @@
...
+  </div>
+  <div class="group-info-content">
+    <alg-pending-join-requests [showSwitch]="false"></alg-pending-join-requests>
+  </div>
+  <div class="group-table bg-white">
+    <alg-section i18n-label label="The teams you joined" icon="fa fa-users">
+      <alg-joined-group-list></alg-joined-group-list>
+    </alg-section>
+  </div>
+  <div class="group-table bg-white">
```
Comment (smadbe) [inline src/app/modules/group/pages/my-groups/my-groups.component.html:17]: The white background should be continuous for these 2 tables.

File: src/app/modules/group/pages/my-groups/my-groups.component.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    .group-title {
+      font-size: 2rem;
+      margin-bottom: 1rem;
+      color: #4A4A4A;
+    }
+
+    .group-subtitle {
+      font-size: 1.2rem;
+    }
```
Comment (smadbe) [inline src/app/modules/group/pages/my-groups/my-groups.component.scss:19]: Quite common to almost all page, could you try to make it reusable?

**Review by smadbe — CHANGES_REQUESTED** [review]

File: src/app/modules/group/pages/my-groups/my-groups.component.scss (hunk 3 above)
Comment (Iloveall) [reply src/app/modules/group/pages/my-groups/my-groups.component.scss]: I have created page-header scss component and it was integrated only for my groups page. Integration it for whole app probably requires separated task

File: src/app/modules/group/pages/my-groups/my-groups.component.html (hunk 2 above)
Comment (Iloveall) [reply src/app/modules/group/pages/my-groups/my-groups.component.html]: Fixed

File: src/app/modules/group/pages/my-groups/my-groups.component.html (hunk 1 above)
Comment (Iloveall) [reply src/app/modules/group/pages/my-groups/my-groups.component.html]: Fixed

**Review by smadbe — APPROVED** [review]
👍

**Pull request merged**